src/gfwapiclient/resources/bulk_downloads/query/models/fixed_infrastructure_data/response.py

```diff
@@ -0,0 +1,133 @@
+]
+
+
+class BulkFixedInfrastructureDataQueryItem(BulkReportQueryItem):
+    """Result item for the fixed infrastructure data dataset.
+
+    Represents a data record of a previously created fixed infrastructure data (i.e.,
+    `public-fixed-infrastructure-data:latest` dataset) bulk report.
+
+    For more details on the Query Bulk Report API endpoint supported response bodies,
+    please refer to the official Global Fishing Watch API documentation:
+
+    See: https://globalfishingwatch.org/our-apis/documentation#get-data-in-json-format
+
+    See: https://globalfishingwatch.org/our-apis/documentation#get-data-in-json-format-http-response
+
+    Attributes:
+        detection_id (Optional[str]):
+            Unique identifier (ID) of the satellite detection (e.g.,
+            `"1AB_AD_MEDIAN_COMP"`).
+
+        detection_date (Optional[datetime.datetime]):
+            Date of the detection (e.g., `"2021-07-01"`).
+
+        structure_id (Optional[Union[str, int]]):
+            Unique identifier (ID) for all detections of the same structure (e.g.,
+            `"162013"`).
+
+        lat (Optional[float]):
+            Latitude of the structure (e.g., `-151.608786096245`).
+
+        lon (Optional[float]):
+            Longitude of the structure (e.g., `60.8646485096125`).
+
+        structure_start_date (Optional[datetime.datetime]):
+            The first date the structure was detected (e.g., `"2017-01-01"`).
+
+        structure_end_date (Optional[datetime.datetime]):
+            The last date the structure was detected (e.g., `"2021-10-01"`).
+
+        label (Optional[str]):
+            Predicted structure type: `oil`, `wind`, or `unknown` (e.g., `"oil"`).
+
+        label_confidence (Optional[str]):
+            Label classification confidence level: `high`, `medium`, or `low` (e.g., `"high"`).
+    """
+
+    detection_id: Optional[str] = Field(None, alias="detection_id")
+    detection_date: Optional[datetime.datetime] = Field(None, alias="detection_date")
+    structure_id: Optional[Union[str, int]] = Field(None, alias="structure_id")
+    lat: Optional[float] = Field(None, alias="lat")
+    lon: Optional[float] = Field(None, alias="lon")
+    structure_start_date: Optional[datetime.datetime] = Field(
+        None, alias="structure_start_date"
+    )
+    structure_end_date: Optional[datetime.datetime] = Field(
+        None, alias="structure_end_date"
+    )
+    label: Optional[str] = Field(None, alias="label")
+    label_confidence: Optional[str] = Field(None, alias="label_confidence")
+
+    @field_validator(
+        "detection_date",
+        "structure_start_date",
+        "structure_end_date",
+        mode="before",
+    )
+    @classmethod
+    def empty_datetime_str_to_none(cls, value: Any) -> Optional[Any]:
+        """Convert any empty datetime string to `None`.
+
+        Args:
+            value (Any):
+                The value to validate.
+
+        Returns:
+            Optional[datetime.datetime]:
```

Comment:
The return type from the typehint is not he same as the one in the docstring. 

An alternative to duplicate the types is just declare them in typehints but not in the docstrings (tools for autodocument the API should be able to pick them from typehints).